firebase-installations/src/androidTest/java/com/google/firebase/installations/FirebaseInstallationsInstrumentedTest.java

```diff
@@ -291,6 +292,31 @@ public void testGetId_PersistedInstallationOk_BackendError() throws Exception {
     assertThat(updatedInstallationEntry).hasRegistrationStatus(RegistrationStatus.REGISTER_ERROR);
   }
 
+  @Test
+  public void testGetId_500ServerError_UnregisteredFID() throws Exception {
+    when(backendClientReturnsOk.createFirebaseInstallation(
```

Comment:
where does this test stub a 500 error?

Reply:
Sorry for the confusion. Added a comment & changed the test name. Does this makes sense?